frontend/src/app/modules/result-selection/components/result-selection-time-frame/result-selection-time-frame.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="card form-horizontal">
```

Comment:
please differ between aggregation charts and time series chart

Reply:
You can now specify whether the aggregation is to be displayed.